templates/taxi/car_detail.html

```diff
@@ -3,20 +3,36 @@
 {% block content %}
   <h1>
     {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
+  </h1>
+  <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
+  <h1>Drivers
+    {% if user in car.drivers.all %}
+      <a href="{% url 'taxi:car-delete-me' car.pk %}" class="btn btn-danger link-to-page">Delete me from this car</a>
```

Comment:
Add a  form here instead  so it can send a POST request to assign  driver in CarDetailView